api/envoy/config/filter/http/rbac/v2/rbac.proto

```diff
@@ -12,18 +12,16 @@ import "gogoproto/gogo.proto";
 // Role-Based Access Control :ref:`configuration overview <config_http_filters_rbac>`.
 
 message RBAC {
-  // Specify the RBAC rules to be applied globally
-  config.rbac.v2alpha.RBAC rules = 1 [(validate.rules).message.required = true];
+  // Specify the enforcement mode RBAC rules to be applied globally.
+  // Disable the filter for enforcement mode if rules isn't set.
+  config.rbac.v2alpha.RBAC rules = 1;
+
+  // Specify the permissive mode RBAC rules to be applied globally.
+  // Disable the filter for permissive mode if permissive_rules isn't set.
+  config.rbac.v2alpha.RBAC permissive_rules = 2;
 }
 
 message RBACPerRoute {
-  oneof override {
-    option (validate.required) = true;
-
-    // Disable the filter for this particular vhost or route.
-    bool disabled = 1 [(validate.rules).bool.const = true];
-
-    // Override the global configuration of the filter with this new config.
-    RBAC rbac = 2 [(validate.rules).message.required = true];
```

Comment:
this is a breaking change. The rbac field must remain tag number 2, and tag 1 and "disabled" must be [reserved](https://developers.google.com/protocol-buffers/docs/proto3#reserved) on the proto message.

Reply:
done.